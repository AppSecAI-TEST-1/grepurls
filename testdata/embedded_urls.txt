[markdown][http://www.google.com]
<markdown><http://www.wiki.org>
Email{mailto:divye@divye.me}
Print("ftp://divye:password@fake.me/")
print('mailto:divye@divye.com')
print('http://divye@divye.me')
(ftp://ftp.is.co.za/rfc/rfc1808.txt)
href="mailto:divye@divye.in"
href='mailto:divye@divye.xyz'
href="ftp://divye:password@fake.me/"
href='ftp://divye:password@fake.me/'
