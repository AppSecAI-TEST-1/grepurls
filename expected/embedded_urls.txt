http://www.google.com
http://www.wiki.org
mailto:divye@divye.me
ftp://divye:password@fake.me/
mailto:divye@divye.com
http://divye@divye.me
ftp://ftp.is.co.za/rfc/rfc1808.txt
mailto:divye@divye.in
mailto:divye@divye.xyz
ftp://divye:password@fake.me/
ftp://divye:password@fake.me/
